File: httpsnippet/src/test/resources/output/node/native/full.js
var qs = require("querystring");
var http = require("http");

var options = {
  "method": "POST",
  "hostname": "mockbin.com",
  "port": 80,
  "path": "/har?baz=abc&foo=bar&foo=baz&key=value",
  "headers": {
    "Cookie": "foo=bar; bar=baz",
    "content-type": "application/x-www-form-urlencoded",
    "accept": "application/json"
  }
};

var req = http.request(options, function (res) {
  var chunks = [];

  res.on("data", function (chunk) {
    chunks.push(chunk);
  });

  res.on("end", function () {
    var body = Buffer.concat(chunks);
    console.log(body.toString());
  });
});

req.write(qs.stringify({"foo":"bar"}));
req.end();

File: httpsnippet/src/test/resources/output/go/native/full.go
package main

import (
	"fmt"
	"strings"
	"net/http"
	"io/ioutil"
)

func main() {

	url := "http://mockbin.com/har?baz=abc&foo=bar&foo=baz&key=value"

	payload := strings.NewReader("foo=bar")

	req, _ := http.NewRequest("POST", url, payload)

	req.Header.Add("Cookie", "foo=bar; bar=baz")
	req.Header.Add("content-type", "application/x-www-form-urlencoded")
	req.Header.Add("accept", "application/json")

	res, _ := http.DefaultClient.Do(req)

	defer res.Body.Close()
	body, _ := ioutil.ReadAll(res.Body)

	fmt.Println(res)
	fmt.Println(string(body))

}
